src/js/labels/type.ts

```diff
@@ -0,0 +1,17 @@
+/* Smaller label object for representing labels in a list */
+export type LabelNested = {
+    /* A hex encoded color */
```

Comment:
```suggestion
    /* The hex encoded color */
```
On re-read I don't like my usage of "A" here.  Should be "The" as "A" leaves more ambiguity suggesting there could be more than one property doing the same thing.

Applies for all fields below as well.